### PR TITLE
[Security] Add missing void PHPdoc return types

### PR DESCRIPTION
**Pull request by mbuliard**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| Tickets       | contribution to #47551
| License       | MIT
| Doc PR        | -

Add missing void PHPdoc return types.

Not sure if in the right place, but remove return portage of FirewallListenerInterface::authenticate() in WrappedLazyListener::authenticate(), since the return is supposed to be void.

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Debug/WrappedLazyListener.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function authenticate(RequestEvent $event)
         $startTime = microtime(true);
 
         try {
-            $ret = $this->listener->authenticate($event);
+            $this->listener->authenticate($event);
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/SecurityBundle/Debug/WrappedLazyListener.php:46]: we should add a native void return type to this method

File: src/Symfony/Component/Security/Core/Authentication/RememberMe/TokenProviderInterface.php [hunk 2]
```diff
@@ -31,18 +31,24 @@ public function loadTokenBySeries(string $series);
...
+     * @return void
      */
     public function deleteTokenBySeries(string $series);
 
     /**
      * Updates the token according to this data.
      *
      * @throws TokenNotFoundException if the token is not found
+     *
+     * @return void
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Core/Authentication/RememberMe/TokenProviderInterface.php:42]: see fabbot, return should be before throws

**Review by nicolas-grekas — APPROVED** [review]
As a rule of thumb, `.github/expected-missing-return-types.diff` should contain no `Tests` classes. Instead, native types should be used for them. I've fixed the PR.

**Comment by nicolas-grekas** [comment]
Thank you @mbuliard.

**Comment by mbuliard** [comment]
> Thank you @mbuliard.

Thank you for the review and corrections

**Pull request merged**